src/app/layout.tsx

```diff
@@ -4,6 +4,7 @@ import { Geist, Geist_Mono } from "next/font/google";
 import { AuthProvider } from "@/context/AuthContext";
 
 import "./globals.css";
+import ResponsiveLayout from "@/components/responsiveLayout";
```

Comment:
Aqui podemos deixar mais acima na linha 5
```suggestion
import { AuthProvider } from "@/context/AuthContext";
import ResponsiveLayout from "@/components/responsiveLayout";

import "./globals.css";
```